include/aws_iot_mqtt_client.h

```diff
@@ -312,112 +312,181 @@ struct _Client {
 };
 
 /**
- * @brief What is the next available packet Id
+ * @functionpage{aws_iot_mqtt_get_next_packet_id,mqtt,get_next_packet_id}
+ * @functionpage{aws_iot_mqtt_set_connect_params,mqtt,set_connect_params}
+ * @functionpage{aws_iot_mqtt_is_client_connected,mqtt,is_client_connected}
+ * @functionpage{aws_iot_mqtt_get_client_state,mqtt,get_client_state}
+ * @functionpage{aws_iot_is_autoreconnect_enabled,mqtt,is_autoreconnect_enabled}
+ * @functionpage{aws_iot_mqtt_set_disconnect_handler,mqtt,set_disconnect_handler}
+ * @functionpage{aws_iot_mqtt_autoreconnect_set_status,mqtt,autoreconnect_set_status}
+ * @functionpage{aws_iot_mqtt_get_network_disconnected_count,mqtt,get_network_disconnected_count}
+ * @functionpage{aws_iot_mqtt_reset_network_disconnected_count,mqtt,reset_network_disconnected_count}
+ */
+
+/**
+ * @brief Retrieve and increment the next packet identifier for an MQTT client context.
+ *
+ * This function generates a two-byte packet identifier for an outgoing MQTT packet and
+ * modifies the internal state of the MQTT client context so that the next call generates
+ * a different packet identifier. Per the MQTT specification, MQTT packet identifiers are
+ * nonzero, two-byte integers that identify certain MQTT packets. MQTT packet identifiers
+ * must be unique at any given time: no two concurrent packets may use the same identifier,
+ * but packet identifiers from previously processed packets may be reused.
  *
- * Called to retrieve the next packet id to be used for outgoing packets.
- * Automatically increments the last sent packet id variable
+ * @param[in] pClient MQTT client context
```

Comment:
(Nit) _The_ MQTT client context *.*

Reply:
In this version, it seems the parameter names don't have periods at the end.  To keep it consistent with the rest of the documentation, I have omitted the periods from these updates.